parse/internal/options/options.go

```diff
@@ -95,12 +95,12 @@ var options = optionMap{
 	ContextLines:      option{noContextLines, "The number of context lines to send to LaunchDarkly. If < 0, no source code will be sent to LaunchDarkly. If 0, only the lines containing flag references will be sent. If > 0, will send that number of context lines above and below the flag reference. A maximum of 5 context lines may be provided.", false},
 	DefaultBranch:     option{"master", "The git default branch. The LaunchDarkly UI will default to this branch.", false},
 	Dir:               option{"", "Path to existing checkout of the git repo. If a cloneEndpoint is provided, this option is not required.", false},
-	Exclude:           option{"", "Exclude any files or directories that match this regular expression pattern", false},
+	Exclude:           option{"", `A regular expression (PCRE) defining the files and directories which the flag parser should exclude. Example: "vendor/"`, false},
 	ProjKey:           option{"", "LaunchDarkly project key.", true},
-	UpdateSequenceId:  option{noUpdateSequenceId, "An integer representing the order number of code reference updates. Used to version updates across concurrent executions of the flag parser. If not provided, data will always be updated. If provided, data will only be updated if the existing `updateSequenceId` is less than the new `updateSequenceId`. Examples: the time a `git push` was initiated, CI build number, the current unix timestamp.", false},
+	UpdateSequenceId:  option{noUpdateSequenceId, `An integer representing the order number of code reference updates. Used to version updates across concurrent executions of the flag parser. If not provided, data will always be updated. If provided, data will only be updated if the existing "updateSequenceId" is less than the new "updateSequenceId". Examples: the time a "git push" was initiated, CI build number, the current unix timestamp.`, false},
```

Comment:
i saw this spit out a really weird result for Alexis. it just said `git push`. just want to confirm thats fixed.

Reply:
I wasn't able to reproduce. Is it possible his terminal wasn't wrapping lines?

Reply:
that is possible. we should check again next secret shopper